lib/validates_zipcode/cldr_regex_collection.rb

```diff
@@ -60,29 +61,30 @@ module CldrRegexpCollection
       IS: /\A\d{3}\z/,
       IN: /\A\d{6}\z/,
       ID: /\A\d{5}\z/,
-      IL: /\A\d{5,7}\z/,
+      IL: /\A\d{5}(\d{2})?\z/,
       JO: /\A\d{5}\z/,
       KZ: /\A\d{6}\z/,
       KE: /\A\d{5}\z/,
       KW: /\A\d{5}\z/,
       LA: /\A\d{5}\z/,
-      LB: /\A(\d{4}([ ]?\d{4})?)?\z/,
-      LU: /\A(L\-)?\d{4}\z/,
+      # NOTE: Lebanon has no postal codes
+      # LB: /\A(\d{4}([ ]?\d{4})?)?\z/,
+      LU: /\A(L|L\-)?\d{4}\z/,
```

Comment:
https://en.wikipedia.org/wiki/Postal_codes_in_Luxembourg

The `L-` is an optional validation so I thought I'd make it slightly more flexible to allow for `L1234` as well as `L-1234`

Our app personally has some data where people have used all three (`1234`, `L1234` and `L-1234`)

I would have thought that this is something that could be looking into with the formatting?